gettsim/transfers/rente.py

```diff
@@ -295,4 +413,447 @@ def ges_rente_regelaltersgrenze(geburtsjahr: int, ges_rente_params: dict) -> flo
+        See :func:`ges_rente_wartezeit_5`.
+
+    Returns
+    -------
+    Eligibility as bool.
+
+    """
+    out = ges_rente_wartezeit_5 >= 5
+
+    return out
+
+
+def ges_rente_vorraus_frauen(
+    weiblich: bool, ges_rente_wartezeit_15: float, jahre_beitr_nach40: float, alter: int
+) -> bool:  # , ges_rente_params: dict
+    """Function determining the eligibility for Altersrente für Frauen (pension
+        for women) Wartezeit 15 years, contributions 10 years after age 40,
+        being a women.
+
+    Parameters
+    ----------
+    weiblich
+        See basic input variable :ref:`weiblich <weiblich>`.
+    ges_rente_wartezeit_15
+        See :func:`ges_rente_wartezeit_15`
+    jahre_beitr_nach40
+        See basic input variable :ref:`jahre_beitr_nach40 <jahre_beitr_nach40>`.
+
+    Returns
+    -------
+    Eligibility as bool.
+
+    """
+    if (
+        weiblich
+        and ges_rente_wartezeit_15 >= 15
+        and jahre_beitr_nach40 >= 10
+        and alter >= 60
+    ):
+        out = True
+    else:
+        out = False
+
+    return out
+
+
+def ges_rente_vorraus_langjährig(ges_rente_wartezeit_35: float, alter: int) -> bool:
+    """Determining the eligibility for Altersrente für langjährig Versicherte
+        (pension for long-term insured). Wartezeit 35 years.
+
+    Parameters
+    ----------
+    ges_rente_wartezeit_35
+        See :func: `ges_rente_wartezeit_35`.
+
+    Returns
+    -------
+    Eligibility as bool.
+
+    """
+    if alter >= 63:
+        out = ges_rente_wartezeit_35 >= 35
+    else:
+        out = False
+
+    return out
+
+
+def ges_rente_vorraus_besond_lang(ges_rente_wartezeit_45: float) -> bool:
+    """Determining the eligibility for Altersrente für besonders langjährig
+        Versicherte (pension for very long-term insured). Wartezeit 45 years.
+        aka "Rente mit 63".
+
+    Parameters
+    ----------
+    ges_rente_wartezeit_45
+        See :func:`ges_rente_wartezeit_45`
+
+    Returns
+    -------
+    Eligibility as bool.
+
+    """
+
+    out = ges_rente_wartezeit_45 >= 45
+
+    return out
+
+
+def ges_rente_wartezeit_5(
+    pflichtbeitragszeit: float, freiw_beitragszeit: float, ersatzzeit: float
+) -> float:
+    """Aggregates time periods that are relevant for the general eligibility
+    of the regular pension (regelaltersrente). "Allgemeine Wartezeit".
+
+    Parameters
+    ----------
+    pflichtbeitragszeit
+        See basic input variable :ref:`pflichtbeitragszeit <pflichtbeitragszeit>`.
+    freiw_beitragszeit
+        See basic input variable :ref:`freiw_beitragszeit <freiw_beitragszeit>`.
+    ersatzzeit
+        See basic input variable :ref:`ersatzzeit <ersatzzeit>`.
+
+    Returns
+    -------
+    Wartezeit in years.
+    """
+    out = (pflichtbeitragszeit + freiw_beitragszeit + ersatzzeit) / 12
+
+    return out
+
+
+# todo maybe implement default  for zeiten if vars
+#  not available? somewhere swtich for eligibility in interface?
+
+
+def ges_rente_wartezeit_15(
```

Comment:
can be combined with `ges_rente_wartezeit_5`

Reply:
better not as "Wartezeit 5 Jahre" is a technical term